.midje.clj

```diff
@@ -0,0 +1 @@
+(change-defaults :fact-filter (complement :thumbnail-integration))
```

Comment:
It's midje's configuration for fact filtering. Is says that the default should not include that key. This is so those integration tests won't run in Travis.